osu.Game/Overlays/Chat/DrawableChannel.cs

```diff
@@ -79,6 +82,36 @@ private void load()
             Channel.PendingMessageResolved += pendingMessageResolved;
         }
 
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+
+            highlightedMessage = Channel.HighlightedMessage.GetBoundCopy();
+            highlightedMessage.BindValueChanged(_ => processMessageHighlighting(), true);
```

Comment:
```suggestion
            highlightedMessage.BindValueChanged(_ => Scheduler.AddOnce(processMessageHighlighting), true);
```

Probably?

Reply:
The method is already scheduled, since all usages are not safe to call it directly. I guess it's better to schedule at usages rather than in the method itself?

Reply:
yeah i saw that. was more of a suggestion to avoid it running more than once, but i guess it's not the end of the world if it does.